data/en/xmlsearch.json

```diff
@@ -4,20 +4,27 @@
 	"syntax":"XMLSearch(xmldoc, xpathstring)",
 	"returns":"array",
 	"related":[],
-	"description":" ",
+	"description":"Get XML values according to given XPath",
 	"params": [
 		{"name":"xmldoc","description":"XML document object","required":true,"default":"","type":"Object","values":[]},
 		{"name":"xpathstring","description":"XPath expression","required":true,"default":"","type":"String","values":[]}
-
 	],
 	"engines": {
 		"coldfusion": {"minimum_version":"", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-t-z/xmlsearch.html"},
 		"lucee": {"minimum_version":"", "notes":"", "docs":"http://docs.lucee.org/reference/functions/xmlsearch.html"},
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/function/xmlsearch"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/xmlsearch"}
 	},
-	"links": [
-
-	]
-
+	"links": [{
+			"title":"Title of a blog entry that has good info about this.",
+			"description":"Description of the link",
+			"url":"http://www.example.com/a/b.cfm"
+	}],
```

Comment:
The link above is an example, did you mean to update that to something else?

Reply:
I wonder, cause I thought I did.